howto: add auto-complete example

kenhys opened the pull request.

kenhys commented:
@kou Could you review this PR?

kou commented on how-to/index.md:
```diff
@@ -8,6 +8,10 @@ This document describes about useful information for specific situations.
 
 If you have useful information that isn't listed in this page, please send a pull request to [pgroonga/pgroonga.github.io](https://github.com/pgroonga/pgroonga.github.io). Please share your useful information!
 
+## How to use auto complete feature with PGroonga
```
"with PGroonga" is needless.

kenhys replied:
OK, I'll fix it.

kou commented on how-to/autocomplete.md:
```diff
@@ -0,0 +1,129 @@
+
+## Use prefix romaji katakana search against auto complete candidate readings
+
+Here is the sample data for RK search.
```
Searching with romaji example is missing.

kenhys replied:
Thanks. I've missed it.

kou commented on how-to/autocomplete.md:
```diff
@@ -0,0 +1,129 @@
+
+# How to use auto complete feature with PGroonga
+
+PGroonga can't use Groonga's suggest plugin directly, so in this
```
We don't fold long lines in this site.

kenhys replied:
ditto

kenhys commented:
I've fixed PR with some feedback.

* removed needless "with PGroonga" from sentence (index.md)
* fixed folded long line (follow style with other sentences) (autocomplete.md)
* added prefix RK search (Romaji) example (autocomplete.md)
* fixed formatted SQL result (added prefix "--" for them) (autocomplete.md)
